SPECS/mesa-libGLU/mesa-libGLU.spec

```diff
@@ -13,7 +13,6 @@ BuildRequires:  autoconf
 BuildRequires:  automake
 BuildRequires:  libtool
 BuildRequires:  mesa-libGL-devel
-#Requires:
```

Comment:
One more request to add that `= %{version}-%{release}` in `Provides: libGLU-devel` a few lines below.